contracts/nft/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "astroport-nft"
+version = "1.0.0"
+authors = ["Astroport"]
+edition = "2021"
+description = "Expanded implementation of cw721 NFTs compliant token for the Astroport NFT"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/cw-nfts"
+homepage = "https://cosmwasm.com"
+documentation = "https://docs.cosmwasm.com"
```

Comment:
```suggestion
```

Reply:
Below, you suggest to marking Cosmwasm as author, so let's leave that information in the package as we did in astroport-token.

Reply:
Agree